src/Sylius/Behat/Resources/config/services/contexts/api/email.xml

```diff
@@ -18,6 +18,7 @@
             <argument type="service" id="sylius.behat.shared_storage" />
             <argument type="service" id="sylius.behat.email_checker" />
             <argument type="service" id="translator" />
+            <argument type="service" id="filesystem" />
```

Comment:
It can be removed, once we stop basing on the existence of spool directory, and check against the number of e-mails sent to a given recipient.